From 66d4d7a5590baaccb0a3a418cdba4c51390ca42e Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Mon, 10 Oct 2022 05:08:58 +0000
Subject: [PATCH] Add CCache Check to Environment Setup

---
 build/envsetup.sh | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/build/envsetup.sh b/build/envsetup.sh
index d191211f..a7ebefa5 100644
--- a/build/envsetup.sh
+++ b/build/envsetup.sh
@@ -978,6 +978,23 @@ function setbuildtype() {
     export GAPPS_BUILD_TYPE=$buildtype
 }
 
+# Check and set ccache path on envsetup
+if [ -z ${CCACHE_EXEC} ]; then
+	ccache_path=$(which ccache)
+	if [ ! -z "$ccache_path" ]; then
+		mkdir /mnt/ccache
+		mount --bind ~/.ccache /mnt/ccache
+		export USE_CCACHE=1
+		export CCACHE_COMPRESS=1
+		export CCACHE_EXEC=/usr/bin/ccache
+		export CCACHE_DIR=/mnt/ccache
+		ccache -M 50G -F 0
+		echo "ccache found and CCACHE_EXEC has been set to : $ccache_path"
+	else
+		echo "ccache not found/installed!"
+	fi
+fi
+
 export SKIP_ABI_CHECKS=true
 
 # Override host metadata to make builds more reproducible and avoid leaking info
-- 
2.34.1

