From c2320985062975cb4724692c146fd81f29d03812 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Sun, 23 Jul 2023 11:03:42 +0000
Subject: [PATCH] Nuke adb.secure everywhere

---
 config/alpha.mk | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/config/alpha.mk b/config/alpha.mk
index 9206e8d3..cf769224 100644
--- a/config/alpha.mk
+++ b/config/alpha.mk
@@ -11,21 +11,6 @@ PRODUCT_PACKAGES += \
     MatLog
 endif
 
-# ADB
-ifeq ($(strip $(TARGET_DEFAULT_ADB_ENABLED)), true)
-  PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
-else
-#  ifeq ($(strip $(TARGET_DEFAULT_ADB_ENABLED)), false)
-#    PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
-#  else
-    ifeq ($(TARGET_BUILD_VARIANT),eng)
-      PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
-    else
-      PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
-    endif
-#  endif
-endif
-
 # Dalvik
 PRODUCT_PROPERTY_OVERRIDES += \
     pm.dexopt.boot=verify \
-- 
2.34.1

