From ff9d9cabb008c6bee53826694a4d8fb9d4ad5602 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Thu, 27 Jul 2023 20:50:38 +0000
Subject: [PATCH] Set bootanimation at 720P

---
 config.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.mk b/config.mk
index d83134bb..4bd92b13 100644
--- a/config.mk
+++ b/config.mk
@@ -18,8 +18,8 @@ PRODUCT_PACKAGE_OVERLAYS += vendor/addons/overlay/common
 PRODUCT_ENFORCE_RRO_EXCLUDED_OVERLAYS += vendor/addons/overlay/common
 
 # Bootanimation
-TARGET_SCREEN_WIDTH ?= 1080
-TARGET_SCREEN_HEIGHT ?= 1920
+TARGET_SCREEN_WIDTH ?= 720
+TARGET_SCREEN_HEIGHT ?= 1080
 PRODUCT_PACKAGES += \
     bootanimation.zip
 
-- 
2.34.1

