From e3262ee5a91cabd3e721e4772adb0815c246278d Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Mon, 17 Apr 2023 17:39:25 +0000
Subject: [PATCH] Inherit call phh_gsi.mk

---
 config/common.mk | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/config/common.mk b/config/common.mk
index 7d1715f1..48740994 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -7,6 +7,22 @@ $(call inherit-product-if-exists, external/faceunlock/config.mk)
 
 PRODUCT_BUILD_PROP_OVERRIDES += BUILD_UTC_DATE=0
 
+# PHH GSI
+$(call inherit-product, vendor/lineage/config/phh_gsi.mk)
+
+# AlphaDroid Flags
+ALPHA_BUILD_TYPE := Treble
+ALPHA_BUILD_PACKAGE := vanilla
+LINEAGE_BUILD := Treble
+TARGET_HAS_UDFPS := true
+TARGET_ENABLE_BLUR := true
+TARGET_INCLUDE_MATLOG := false
+TARGET_USE_GRAPHENE_CAMERA := false
+TARGET_USE_PIXEL_LAUNCHER := false
+TARGET_EXCLUDES_AUDIOFX := true
+TARGET_FACE_UNLOCK_SUPPORTED := true
+ALPHA_MAINTAINER := ChonDoe
+
 ifeq ($(PRODUCT_GMS_CLIENTID_BASE),)
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     ro.com.google.clientidbase=android-google
-- 
2.34.1

